common/tools/dev-tool/src/commands/samples/publish.ts

```diff
@@ -326,7 +326,7 @@ async function makeSampleGenerationInfo(
     return undefined as never;
   }
 
-  const moduleInfos = await processSources(projectInfo, sampleSources, fail);
+  const moduleInfos = await processSources(projectInfo, sampleSources as string[], fail);
```

Comment:
why is this type cast needed?

Reply:
Yeah. I have fixed it. It is redundant after I added `AsyncGenerator<string>` to `common/tools/dev-tool/src/util/findMatchingFiles.ts`